src/components/UsedByItem.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { FC } from "react";
+import ResourceLink from "./ResourceLink";
+import { LxdUsedBy } from "util/usedBy";
+import { ResourceIconType } from "./ResourceIcon";
+import { useQuery } from "@tanstack/react-query";
+import { fetchImageList } from "api/images";
+import { queryKeys } from "util/queryKeys";
+
+interface Props {
+  item: LxdUsedBy;
+  activeProject: string;
+  type: ResourceIconType;
+  to: string;
+}
+
+const UsedByItem: FC<Props> = ({ item, activeProject, type, to }) => {
+  const { data: images = [] } = useQuery({
+    queryKey: [queryKeys.images],
+    queryFn: () => fetchImageList(activeProject),
+    enabled: type === "image",
+  });
+
+  const image = images.find((image) => image.fingerprint === item.name);
+
+  let label = item.name;
+  if (type === "image") {
+    label = image?.properties?.description || item.name;
+  }
```

Comment:
```suggestion
  const label = image?.properties?.description || item.name;
```
Can simplify this to one line.